src/assets/scripts/client/info/AirportInfoView.js

```diff
@@ -0,0 +1,162 @@
+import $ from 'jquery';
+import EventBus from '../lib/EventBus';
+import SimClockController from './SimClockController';
+import AirportInfoController from './AirportInfoController';
+import { EVENT } from '../constants/eventNames';
+import { SIM_AIRPORT_INFO_TEMPLATE } from './simAirportInfoTemplate';
+
+/**
+ * Displays an information corner that displays the current time, wind speed,
+ * wind angle, altimeter, and airport elevation.
+ *
+ * @class AirportInfoView
+ */
+export default class AirportInfoView {
+    /**
+     * @for AirportInfoView
+     * @constructor
+     * @param {jQuery|HTML element}
+     */
+    constructor($element) {
+        /**
+         * Root DOM element
+         *
+         * @for AirportInfoView
+         * @property $element
+         * @type {jQuery|HTML element}
+         */
+        this.$element = $element;
+
+        /**
+         * Information div
+         *
+         * @for AirportInfoView
+         * @property $template
+         * @type {jQuery|HTML element}
+         */
+        this.$template = null;
+
+        /**
+         * @for AirportInfoView
+         * @property simClockController
+         */
+        this.simClockController = null;
+
+        /**
+         * @for AirportInfoView
+         * @property simAirportInfoController
+         */
+        this.simAirportInfoController = null;
+
+        /**
+         * Local instance of the event bus
+         *
+         * @for AirportInfoView
+         * @property _eventBus
+         * @type {EventBus}
+         */
+        this._eventBus = EventBus;
+
+        return this.init();
+    }
+
+    /**
+     * @for AirportInfoView
+     * @method init
+     * @chainable
+     */
+    init() {
+        this.$template = $(SIM_AIRPORT_INFO_TEMPLATE);
+        this.simClockController = new SimClockController();
+        this.simAirportInfoController = new AirportInfoController();
+
+        this._setupHandlers();
+        this._eventBus.on(EVENT.AIRPORT_CHANGE, this.onAirportChangeHandler);
```

Comment:
this should move to a chainable method named `_enable()`.  there should also be an `.off` version of this in another new method called `._disable()`